__tests__/storage/storage.test.js

```diff
@@ -143,3 +143,230 @@ test('utxos selected as inputs', async () => {
   await storage.utxoSelectAsInput({ txId: 'a-tx-id3', index: 0 }, true);
   expect(storage.utxosSelectedAsInput.get('a-tx-id3:0')).toBeUndefined();
 });
+
+test('process locked utxo', async () => {
+  const nowTs = Math.floor(Date.now() / 1000);
+  const tsLocked = nowTs + 60;
+  const tsUnLocked = nowTs - 60;
+
+  function getLockedUtxo(txId, address, timelock, height, value, token, token_data) {
+    return {
+      index: 0,
+      tx: {
+        tx_id: txId,
+        height,
+        version: 1,
+        timestamp: timelock,
+        is_voided: false,
+        inputs: [],
+        outputs: [
+          {
+            value,
+            token_data,
+            token,
+            spent_by: null,
+            decoded: {
+              type: 'P2PKH',
+              address,
+              timelock,
+            }
+          },
+        ],
+      },
+    };
+  }
+
+  function getUtxoFromLocked(lutxo) {
+    const { tx, index } = lutxo;
+    const { outputs } = tx;
+    const output = outputs[index];
+    const { decoded } = output;
+    const { address, timelock } = decoded;
+    return {
+      txId: tx.tx_id,
+      index,
+      token: output.token,
+      address,
+      value: output.value,
+      authorities: 0,
+      timelock,
+      type: tx.version,
+      height: tx.height,
+    };
+  }
+
+  const store = new MemoryStore();
```

Comment:
Can we do the same test for leveldb store?

Reply:
Done